src/utils/canvas.ts

```diff
@@ -0,0 +1,26 @@
+import { MimeType } from 'src/layer/const';
+
+export async function drawBlobOnCanvas(
```

Comment:
Would it be better for `drawBlobOnCanvas` to get the `canvas`, not just its `context`? (would be more in line with `canvasToBlob()`)

Reply:
At least how it is used in `getHugeMap`, this is more convenient because we can create one huge canvas and then draw the blobs on it. We can however later add another function (`blobToCanvas`?) if needed, if you agree?

Reply:
I agree. Resolving